components/compliance-service/api/tests/60_A1_migration_spec.rb

```diff
@@ -31,7 +31,8 @@
           "name": "centos6-node1",
           "platform": {
             "name": "mac_os_x",
-            "release": "10.12.6"
+            "release": "10.12.6",
```

Comment:
All the ruby migration tests need to be updated because they all migrated from their version to the current version.